src/java/org/apache/cassandra/cql3/selection/Selectable.java

```diff
@@ -224,9 +224,9 @@ public static class WritetimeOrTTL implements Selectable
     {
         public enum Kind
         {
+            TTL("ttl", Int32Type.instance),
             WRITE_TIME("writetime", LongType.instance),
-            MAX_WRITE_TIME("maxwritetime", LongType.instance),
-            TTL("ttl", Int32Type.instance);
+            MAX_WRITE_TIME("maxwritetime", LongType.instance);
```

Comment:
Perhaps we should add a comment mentioning that versions up to 4.1 don't have `MAX_WRITE_TIME`.

Reply:
And/or maybe a comment telling that the order of the enum matters for serialization, so we or others don't repeat the same mistake if we modify this again?